windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,38 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.1
+
+Issued:2021-06-23
+
+The WMCO 2.0.1 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-2021:2130[RHSA-2021:2130].
+
+[id="wmco-2-0-1-improvements"]
+=== New features and improvements
+
+This release adds the following new features and improvements.
+
+[id="wmco-2-0-1-image-timeout"]
+==== Increased image pull time-out duration
+
+Image pull time-out has been increased to 30 minutes.
+
+[id="wmco-2-0-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, when using the Windows kube-proxy component on an AWS installation, when you create a LoadBalancer service, packets would be misrouted and reach an unintended destination. Now, you can set the `.status.loadBalancer.ingress.ip` to ensure packets are no longer wrongly routed to unintended destinations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946538[**BZ#1946538**])
+
+* Previously, Windows nodes were not reporting some key node level metrics via telemetry monitoring. The `windows_exporter` reports various metrics as `windows_*` instead of the `node_exporter` equivalent of `node_*`. Now, the telemetry results cover all of the expected metrics. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955319[**BZ#1955319**])
+
+* While the WMCO configures Windows instances, if the hybrid-overlay or kube-proxy components fail, then the node will show as not ready. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1956412[**BZ#1956412**])
+
+* Previously, the kube-proxy service would terminate unexpectedly after the load balancer is created if you create the load balancer after the Windows pods begin running. The kube-proxy service does not crash when recreating the load balancer service. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939968[**BZ#1939968**])
+
+=== RHSA-2021:2130 - Windows Container support for {product-title} security update
+
+An update for Windows kube-proxy is now available for Red Hat Windows Machine Config Operator 2.0.1. Details of the update are documented in the link:https://access.redhat.com/errata/RHSA-2021:2130[RHSA-2021:2130] advisory.
```

Comment:
Should this be connected to https://bugzilla.redhat.com/show_bug.cgi?id=1946538 mentioned in `Bug fixes`? Just want to call out that the fix for the bug and the CVE are one and the same.